web/auth/guardian_serializer.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CodeCorps.GuardianSerializer do
+  @behaviour Guardian.Serializer
+
+  alias CodeCorps.Repo
+  alias CodeCorps.User
+
+  def for_token(user = %User{}), do: { :ok, "User:#{user.id}" }
+  def for_token(_), do: { :error, "Unknown resource type" }
+
+  def from_token("User:" <> id), do: { :ok, Repo.get(User, id) }
```

Comment:
I'm not sure. If you can decode the JWT, then you were able to successfully authenticate the user and generate the JWT in the first place. Unless we're worried that a user may get deleted from the system after they've authenticated to the system, I don't know if it's necessary.

FWIW, I think the Guardian docs are just kind of showing what's possible with the `from_token/1` method.